.tekton/push.yaml

```diff
@@ -105,9 +122,47 @@ spec:
               secretName: redhat-appstudio-tekton-catalog-build-definitions-pull-secret
           workspaces:
             - name: source
+
+      - name: build-bundles-konflux-ci
+        params:
+          - name: revision
+            value: "$(params.revision)"
+        runAfter:
+          - build-container
+        workspaces:
+          - name: source
+            workspace: workspace
+        taskSpec:
+          params:
+            - name: revision
+              type: string
+          steps:
+            - name: build-bundles
+              image: quay.io/konflux-ci/appstudio-utils:{{ revision }}
+              # per https://kubernetes.io/docs/concepts/containers/images/#imagepullpolicy-defaulting
```

Comment:
This is one of those comments that just keeps getting copied around. I think it provides very little value and we should stop propagating the noise. :smile:

Reply:
Removed